.travis.yml

```diff
@@ -16,20 +16,23 @@ matrix:
   fast_finish: true
   include:
     - php: 7.2
+
+    - php: 7.3
+      env: ACTION="update" STABILITY="beta" SYMFONY="5.*"
     - php: 7.3
-      env: SYMFONY="5.0.*"
-           ACTION="update"
+      env: ACTION="update" STABILITY="dev" SYMFONY="5.*"
+
     - php: 7.4
 
 before_install:
-  - '[[ "$TRAVIS_PHP_VERSION" == "7.4snapshot" ]] || phpenv config-rm xdebug.ini'
+  - phpenv config-rm xdebug.ini || true
   - composer self-update
-  # Set memory to max (memory fail)
-  - '[[ "$ACTION" == "install" ]] || echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini'
-  # Set stability to dev to allow 4.4dev and 5.0dev
-  - '[[ "$ACTION" == "install" ]] || composer config minimum-stability dev'
-  # Change version of symfony when need
-  - '[[ "$ACTION" == "install" ]] || composer config extra.symfony.require $SYMFONY'
+  # Set memory to unlimited for composer update
+  - [[ $ACTION == "install" ]] || echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```

Comment:
you could simplify the CI config by using the [`COMPOSER_MEMORY_LIMIT` env variable](https://getcomposer.org/doc/03-cli.md#composer-memory-limit) instead.

Reply:
I did add the COMPOSER_MEMORY_LIMIT env variable